controllers/productController.js

```diff
@@ -0,0 +1,18 @@
+const productService = require('../services/productService')
+
+const reviewService = require('../services/reviewService')
+
+const getProductDetail = async (req, res) => {
+    try {
+        const productId = req.params.productId 
+        const [product] = await productService.getProduct(productId);
+        if(product === undefined) return res.status(404).json({message:'PRODUCT_NOT_FOUND'});
+        const review = await reviewService.findReview(productId);
+        if(review[0].id === null) return res.status(200).json({product});
+        res.status(200).json({product, review});
+    } catch (err) {
+        console.error(err);
+    }
+}
```

Comment:
```suggestion
        if(review[0].id === null) return res.status(200).json({product});
        res.status(200).json({product, review});
    } catch (err) {
        console.error(err);
        res.status(err.statusCode || 500).json({message: err.message})
    }
}
```
에러 나면 프론트에게 아무 응답도 안보내는 상황으로 개발되어 있습니다.
에러나면 프론트에게 에러 상태코드와 함께 메세지 같이 전송하셔야 합니다